### PR TITLE
add CI similar to pymc

### LINKED ISSUE #17: Enable tests on Github

### PR DESCRIPTION
**Pull request by aloctavodia**
Closes #17

### CODE REVIEW

File: conda-envs/environment-test-py39.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: pymc-test-py39
```
Comment (twiecki) [inline conda-envs/environment-test-py39.yml:1]: maybe let's just test on py39 and not the others.

File: conda-envs/environment-test-py39.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+channels:
+- conda-forge
+- defaults
+dependencies:
+- pymc>=4.0.0b1
```
Comment (twiecki) [inline conda-envs/environment-test-py39.yml:6]:
```suggestion
- pymc>=4.0.0b3
```

File: conda-envs/environment-test-py39.yml (hunk 1 above)
Comment (aloctavodia) [reply conda-envs/environment-test-py39.yml:1]: I dropped 3.7 (we should also do it in PyMC, following numpy schedule). We should try to keep the test here and in pymc (approximately) in sync. As that will help to reduce the friction of moving code from here to pymc. And possible reduce issues with users using both pymc and pymc-experimental.

Comment (twiecki) [reply conda-envs/environment-test-py39.yml:1]: Agreed on all points.

**Pull request merged**